pom.xml

```diff
@@ -35,6 +35,7 @@
 		<module>test-embedded-kafka</module>
 		<module>kinesis-produce-consume</module>
 		<module>kstream</module>
+		<module>testing</module>
```

Comment:
@artembilan `testing` sounds very generic. Shall we call the artifact something else? for ex - `spring-cloud-stream-apps-testing` or something along the lines?

Reply:
Well, I won't mind, indeed, but I just follow the existing state: `sink`, `source`, `double`, `kstream`, `transform` finally...
That really should be obvious by the umbrella `spring-cloud-stream-samples` project that the story is about testing SCSt. OTOH we will have some good README there just on the index page of the module... Also maven artifact has some description as well.

Reply:
Well, sorry for the misunderstanding. Yes, let's rename the artifact! The module I guess may stay with `testing` name.